src/Shared/programs.ts

```diff
@@ -1,85 +1,84 @@
-/** 
+/**
  * Program related info
  */
-import * as Sentry from '@sentry/browser'
-
+import * as Sentry from "@sentry/react"
 
 interface Program {
-    full_name: string
-    slug: string
-    url: string
+  full_name: string
+  slug: string
+  url: string
 }
 
 const programs: Program[] = [
-    {
-        full_name: 'National Estuarine Research Reserve System',
-        slug: 'NERRS',
-        url: 'http://cdmo.baruch.sc.edu/'
-    },
-    {
-        full_name: 'NOAA National Data Buoy Center',
-        slug: 'NOAA',
-        url: 'http://www.ndbc.noaa.gov/'
-    },
-    {
-        full_name: 'NOAA National Data Buoy Center',
-        slug: 'CMAN',
-        url: 'http://www.ndbc.noaa.gov/'
-    },
-    {
-        full_name: 'Coastal Data Information Program',
-        slug: 'CDIP',
-        url: 'http://cdip.ucsd.edu/'
-    },
-    {
-        full_name: 'NERACOOS',
-        slug: 'NERACOOS',
-        url: 'http://neracoos.org/'
-    },
-    {
-        full_name: 'NERACOOS',
-        slug: 'NERACOOS Long Is. Sound',
-        url: 'http://neracoos.org/'
-    },
-    {
-        full_name: 'NERACOOS',
-        slug: 'URI',
-        url: 'http://neracoos.org/'
-    },
-    {
-        full_name: 'NERACOOS',
-        slug: 'COOA',
-        url: 'http://neracoos.org/'
-    },
-    {
-        full_name: 'University of New Hampshire Ocean Process Analysis Laboratory',
-        slug: 'UNH Buoy',
-        url: 'http://www.opal.sr.unh.edu/'
-    },
-    {
-        full_name: 'SmartAtlantic Alliance',
-        slug: 'SmartAtlantic',
-        url: 'http://www.smartatlantic.ca/'
-    },
-    {
-        full_name: 'SmartBay',
-        slug: 'SmartBay',
-        url: 'http://www.smartatlantic.ca/PlacentiaBay/'
-    }
+  {
+    full_name: "National Estuarine Research Reserve System",
```

Comment:
Codacy found an issue: [Identifier 'full_name' is not in camel case.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6083672)